src/main/java/ch/njol/skript/sections/SecConditional.java

```diff
@@ -357,13 +361,8 @@ private static List<SecConditional> getElseIfs(List<TriggerItem> triggerItems) {
 		List<SecConditional> list = new ArrayList<>();
 		for (int i = triggerItems.size() - 1; i >= 0; i--) {
 			TriggerItem triggerItem = triggerItems.get(i);
-			if (triggerItem instanceof SecConditional) {
-				SecConditional secConditional = (SecConditional) triggerItem;
-
-				if (secConditional.type == ConditionalType.ELSE_IF)
-					list.add(secConditional);
-				else
-					break;
+			if (triggerItem instanceof SecConditional secConditional && secConditional.type == ConditionalType.ELSE_IF) {
```

Comment:
```suggestion
			if (triggerItem instanceof SecConditional conditionalSection && secConditional.type == ConditionalType.ELSE_IF) {
```
same as in `getPrecedingConditional`

Reply:
ah no reason for this name in particular but in the method above you use conditionalSection instead of secConditional, meant it for consistency but isn't really important

Reply:
ah gotcha, it's all pre-existing code so i don't know the logic behind the choices